.gitignore

```diff
@@ -0,0 +1,11 @@
+# YAML property files
+credentials.yml
+
+# Temporary files
+tmp
+
+# Mac files
+.DS_Store
```

Comment:
To be pedantic, files like this can (should?) be excluded in a global gitignore and not per-project.